src/test/java/edu/hawaii/its/api/groupings/GroupingMemberTest.java

```diff
@@ -25,8 +25,8 @@
 @SpringBootTest(classes = { SpringBootWebApplication.class })
 public class GroupingMemberTest {
 
-    @Value("${groupings.api.test.uh-usernames}")
-    private List<String> TEST_USERNAMES;
+    @Value("${groupings.api.test.uh-uids}")
+    private List<String> TEST_UIDS;
```

Comment:
Sorry I am having you undo a lot these changes. I am trying to reduce the number of changes and split the remaining changes into another ticket. That way any possible merge conflicts are easier to deal with.